.github/workflows/verify-manifest.yml

```diff
@@ -0,0 +1,52 @@
+name: Check Velero Manifest
+
+on:
+  workflow_dispatch:
+
+jobs:
+  check-velero-manifest:
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v3
+
+      - name: Create kind cluster
+        uses: helm/kind-action@v1.5.0
+
+      - name: Install velero
+        run: |
+          wget https://github.com/vmware-tanzu/velero/releases/download/v1.11.1/velero-v1.11.1-linux-amd64.tar.gz
+          tar -xvf velero-v1.11.1-linux-amd64.tar.gz
+          cd velero-v1.11.1-linux-amd64
```

Comment:
The velero image tag should comes from the https://github.com/vmware-tanzu/helm-charts/blob/a9dac3e30d6ff7e8ecd049c760eb0a24223ecd00/charts/velero/values.yaml#L9

Because the problem we want to address is the dependabot bump the velero image tag version, and file a PR. Then, we need to make sure the crds/ folders are sync with the bumped velero image tag.